statics/js/components/capture-form.js

```diff
@@ -114,6 +124,18 @@ Vue.component('capture-form', {
 
   computed: {
 
+    options: function() {
+      var options = {};
+      var allowedTypes = ["ovsbridge", "device", "internal", "veth", "tun", "bridge", "dummy", "gre", "bond", "can", "hsr", "ifb", "macvlan", "macvtap", "vlan", "vxlan", "gretap", "ip6gretap", "geneve", "ipoib", "vcan", "ipip", "ipvlan", "lowpan", "ip6tnl", "ip6gre", "sit", "dpdkport"];
```

Comment:
could be nice to use a common list as this one is already defined in node_selector.js, what do you think about moving them to api.js ?

Reply:
we discussed to have an API to retrieve this list from common.go in another patch, so temporarily i defined here. if you want me this to move to api.js, i am ok with it.

Reply:
right, but meanwhile I think it is better to have one definition

Reply:
sure Sylvain will do.